src/llmcompressor/transformers/finetune/data/data_args.py

```diff
@@ -154,13 +154,13 @@ class DataTrainingArguments(CustomDataTrainingArguments):
         },
     )
     min_tokens_per_module: Optional[float] = field(
-        default=0.2,
+        default=None,
         metadata={
             "help": (
                 "The minimum percentage of tokens (out of the total number) "
                 "that the module should 'receive' throughout the forward "
                 "pass of the calibration. If a module receives fewer tokens, "
-                "a warning will be logged."
+                "a warning will be logged. Defaults to 1/num_of_experts"
```

Comment:
isn't this default for all models? not just MoE?

Reply:
All models don't have experts so no this applies only to  MOEs

Reply:
sorry I meant you're updating the help for `min_tokens_per_module` in `DataTrainingArguments`  - wouldn't these arguments be used for cases that are not MoEs?

Reply:
Syncing offline